serviceWrappers/linux/kLiveController

```diff
@@ -23,36 +23,55 @@
 ###
 source /etc/rc.d/init.d/functions
 
+LOG_DIR=@LOG_DIR@
+PID_DIR=@RUN_DIR@
+USE_NVM=@USE_NVM@
+KLIVE_CONTROLLER_PREFIX="@KLIVE_CONTROLLER_PREFIX@"
+
+#default values
+if [ "${LOG_DIR}" = "@LOG_DIR@" ]; then
+    LOG_DIR=/var/log
+fi
+if [ "${PID_DIR}" = "@RUN_DIR@" ]; then
+    PID_DIR=/var/run
+fi
+if [ "${USE_NVM}" = "@USE_NVM@" ]; then
+    USE_NVM="1"
+fi
+if [ "${KLIVE_CONTROLLER_PREFIX}" = "@KLIVE_CONTROLLER_PREFIX@" ]; then
+    KLIVE_CONTROLLER_PREFIX=/opt/kaltura/liveController/latest
+fi
+
+
 NAME="kLiveController"
-KLIVE_CONTROLLER_PREFIX="/opt/kaltura/liveController/latest"
-NODE_PATH="$KLIVE_CONTROLLER_PREFIX/node_modules"
-FOREVER="$NODE_PATH/forever/bin/forever"
-APPLICATION_PATH="$KLIVE_CONTROLLER_PREFIX/lib/App.js"
-LOG_DIR="/var/log/liveController"
-PID_DIR="/var/run"
+NODE_PATH="${KLIVE_CONTROLLER_PREFIX}/node_modules"
+FOREVER="${NODE_PATH}/forever/bin/forever"
+APPLICATION_PATH="${KLIVE_CONTROLLER_PREFIX}/lib/App.js"
 MIN_UPTIME="5000"
 SPIN_SLEEP_TIME="2000"
-export PATH=$PATH:$NODE_PATH/forever/bin:/usr/local/bin
+export PATH=$PATH:${NODE_PATH}/forever/bin
 HOME=${HOME:-~}
 
 loadNvm() {
-    # This loads nvm
-    # must cd/pushd due to a bug in /root/.nvm/nvm.sh call to nvm_find_nvmrc()  
-    pushd ${KLIVE_CONTROLLER_PREFIX}
-    [ -z "${NVM_DIR}" ] && NVM_DIR="${HOME}/.nvm"
-    if [ -s "${NVM_DIR}/nvm.sh" ]; then
-        source "${NVM_DIR}/nvm.sh"
-        if [ -s ".nvmrc" ] ; then
-            nvm use || nvm install
+    if [ "${USE_NVM}" = "1" ]; then
```

Comment:
Why not check if USE_NVM is set outside of the function and not call it if it isn't?